Packages/Tracking/Core/Editor/Scripts/PrefabCreateMenu.cs

```diff
@@ -0,0 +1,70 @@
+/******************************************************************************
+ * Copyright (C) Ultraleap, Inc. 2011-2021.                                   *
+ *                                                                            *
+ * Use subject to the terms of the Apache License 2.0 available at            *
+ * http://www.apache.org/licenses/LICENSE-2.0, or another agreement           *
+ * between Ultraleap and you, your company or other organization.             *
+ ******************************************************************************/
+
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+
+namespace Leap
+{
+    public class PrefabCreateMenu
+    {
+        [MenuItem("GameObject/Ultraleap/Service Provider (Desktop)", false, 1)]
+        public static void CreateServiceProviderDesktop()
+        {
+            CreatePrefab("Service Provider (Desktop)");
+        }
+
+        [MenuItem("GameObject/Ultraleap/Service Provider (Screentop)", false, 1)]
+        public static void CreateServiceProviderScreentop()
+        {
+            CreatePrefab("Service Provider (Screentop)");
+        }
+
+        [MenuItem("GameObject/Ultraleap/Service Provider (XR)", false, 1)]
+        public static void CreateServiceProviderXR()
+        {
+            CreatePrefab("Service Provider (XR)");
+        }
+
+        public static void CreatePrefab(string prefabName)
+        {
+            GameObject newObject = null;
+
+            if(newObject == null)
```

Comment:
Why is this check needed?

Reply:
oops! this code iterated a lot and I left it in!